app/_src/quickstart/docker-demo.md

```diff
@@ -0,0 +1,716 @@
+
+   ```sh
+   docker run \
+     --detach \
+     --name {{ kuma-demo }}-control-plane \
+     --hostname control-plane \
+     --network {{ kuma-demo }} \
+     --ip {{ ip_abc }}.1 \
+     --publish 25681:5681 \
+     --volume {{ tmp }}:/demo \
+     {{ docker_org }}/kuma-cp:{{ version_full }} run
+   ```  
+
+   You can now access the [{{ Kuma }} user interface (GUI)]({{ docs }}/production/gui/) at <http://127.0.0.1:25681/gui>.
+
+2. **Configure kumactl**
+
+   To use {{ kumactl }} with our {{ Kuma }} deployment, we need to connect it to the control plane we set up earlier.
+
+   1. **Retrieve the admin token**
+
+      Run the following command to get the admin token from the control plane:
+
+      ```sh
+      export {{ KUMA_DEMO_ADMIN_TOKEN }}="$( 
+        docker exec --tty --interactive {{ kuma-demo }}-control-plane \
+          wget --quiet --output-document - \
+          http://127.0.0.1:5681/global-secrets/admin-user-token \
+          | jq --raw-output .data \
+          | base64 --decode
+      )"
+      ```
+
+   2. **Connect to the control plane**
+
+      Use the retrieved token to link {{ kumactl }} to the control plane:
+
+      ```sh
+      kumactl config control-planes add \
+        --name {{ kuma-demo }} \
+        --address http://127.0.0.1:25681 \
+        --auth-type tokens \
+        --auth-conf "token=${{ KUMA_DEMO_ADMIN_TOKEN }}" \
+        --skip-verify
+      ```
+
+   3. **Verify the connection**
+
+      Run this command to check if the connection is working:
+
+      ```sh
+      kumactl get meshes
+      ```
+
+      You should see a list of meshes with one entry: `default`. This confirms the configuration is successful.
+
+3. **Configure the default mesh**
+
+   Set the default mesh to use {{ MeshServices }} in [exclusive mode]({{ docs }}/networking/meshservice/#options). MeshServices are explicit resources that represent destinations for traffic in the mesh. They define which {{ Dataplanes }} serve the traffic, as well as the available ports, IPs, and hostnames. This configuration ensures a clearer and more precise way to manage services and traffic routing in the mesh.
+
+   ```sh
+   echo 'type: Mesh
+   name: default
+   meshServices:
+     mode: Exclusive' | kumactl apply --file -
+   ```
+
+## Set up services
+
+{% capture code-block-install-tools-create-user %}
+```sh
+# install necessary packages
+apt-get update && \
+  apt-get install --yes curl iptables
+
+# download and install {{ Kuma }}
+curl --location {{ url_installer_external }} {% if version == "preview" or page.edition and page.edition != "kuma" %}\
+  {% endif %}| VERSION="{{ version_full }}" sh -
+
+# move {{ Kuma }} binaries to /usr/local/bin/ for global availability
+mv {{ kuma }}-{{ version_full }}/bin/* /usr/local/bin/
+
+# create a dedicated user for the data plane proxy
+useradd --uid 5678 --user-group {{ kuma-data-plane-proxy }}
```

Comment:
Shouldn't all this be in the docker container already (probalby not the dp but the rest)